contracts/mocks/InsecureInvariantTargetBounty.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.4.24;
 // When this line is split, truffle parsing fails.
 // See: https://github.com/ethereum/solidity/issues/4871
 // solium-disable-next-line max-len
-import {BreakInvariantBounty, Target} from "../../contracts/bounties/BreakInvariantBounty.sol";
+import {BreakInvariantBounty, Target} from "../../contracts/drafts/BreakInvariantBounty.sol";
```

Comment:
Note that we changed this in #1319 and #1321, we may want to cherry-pick those commits in the rc.

Reply:
Good idea.